groups/rds/vault.tf

```diff
@@ -0,0 +1,19 @@
+variable "hashicorp_vault_username" {
```

Comment:
This will need to be changed as it's enforcing a username and password requirement on the code. This isn't an approach we'd use in the pipeline as we'd use roles instead. Take a closer look at the [kafka stack](https://github.com/companieshouse/kafka-stack/blob/main/groups/zookeeper/vault.tf) and you'll see that the `vault.tf` file is actually a symlink. If you copy that approach it will be more flexible